plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs

```diff
@@ -308,8 +352,10 @@ private FileModel GenerateNewFileModel(FileModel model, PageViewModel newPage, s
             var initialFile = model.FileAndType.File;
             var extension = Path.GetExtension(initialFile);
             var directory = Path.GetDirectoryName(initialFile);
-            var newFileName = PathUtility.ToValidFilePath(key, '-').Replace('`', '-') + extension;
```

Comment:
convert path to clean url

---
In reply to: [102865868](https://github.com/dotnet/docfx/pull/1341#discussion_r102865868) [](ancestors = 102865868,102860136)